src/nifs/protogalaxy/tests.rs

```diff
@@ -140,7 +140,7 @@ impl<C: Circuit<Scalar>> Mock<C> {
         )
         .expect("`protogalaxy::prove` failed");
 
-        ProtoGalaxy::is_sat_acc(&self.ck, &self.S, &accumulator_from_prove)
+        ProtoGalaxy::is_sat(&self.ck, &self.S, &accumulator_from_prove)
```

Comment:
Besides the sat of accumulator, have you checked the is_sat for `(u,w)`?  At the end, we need check the accumulator is_sat and the non-folded instance is sat.

Reply:
You meen check `PlonkTrace`? I'll add check, but this is not related to this PR. Let's finish with it and I'll add `S.is_sat` to the protogalaxy tests in prev PR